API.md

```diff
@@ -91,6 +91,14 @@ An `HTTP 404 Not Found` status code is returned if the case with the specified I
 ]
 ```
 
+An 'HTTP 204 No Content' status code is returned is the aborted action with the specified ID has been rescheduled.
```

Comment:
this sentence grammatically doesn't make sense. Should be `A` at beginning and missing a when in middle.